.circleci/config.yml

```diff
@@ -48,7 +48,7 @@ commands:
         type: string
       consul-k8s-image:
         type: string
-        default: "docker.mirror.hashicorp.services/hashicorpdev/consul-k8s:latest"
+        default: "kschoche/consul-k8s-dev-fed"
```

Comment:
Will change this  back to normal once we merge the consul-k8s PR.

Reply:
I've done this and pushed it since Kyle's away tomorrow.